troubleshooting.md

```diff
@@ -156,32 +166,11 @@ If you are unable to connect, then troubleshoot the network.
 3. Confirm you can read from the Ceph cluster
 
 If the Ceph cluster is reachable via the network, then try to use the
-`rbd` command to read from it. The `rbd` command is in Nova pods but
-not Glance or Cinder pods. The Nova containers which use the cephx key
-are hosted on Compute nodes, not OpenShift, and thus the NovaPods on
-OpenShift do not have access to the ceph files describe in step 1.
-
-First update the `extraMounts` option described in the
-[Ceph integration documentation](ceph.md) so that the propagation
-list includes the NovaAPI service.
-
-```
-spec:
-  ...
-  extraMounts:
-    - name: v1
-      region: r1
-      extraVol:
-        - propagation:
-          - CinderVolume
-          - GlanceAPI
-          - NovaAPI
-```
-The above update will restart the NovaAPI pod. Connect to the
-restarted pod using `oc rsh` and confirm the required keys
-are present as described in step 1.
-
-Once connected to the NovaAPI pod with `oc rsh` examine the cephx key.
+`rbd` command to read from it.
+
+**Note**
+
+Before running the rbd command, examine the cephx key.
```

Comment:
The command below on line 189 and 190 needs back slashes.

When I run it as written it doesn't parse.

Also on line 222 please remove the `^C`.

Reply:
Done